api/groups/baseVmValuesGroup.go

```diff
@@ -136,11 +143,25 @@ func createSCQuery(request *VMValueRequest) (*data.SCQuery, error) {
 	}, nil
 }
 
+func extractBlockCoordinates(context *gin.Context) (core.OptionalUint64, []byte, error) {
+	blockNonce, err := parseUint64UrlParam(context, common.UrlParameterBlockNonce)
+	if err != nil {
+		return core.OptionalUint64{}, nil, fmt.Errorf("%w for block nonce", err)
+	}
+
+	blockHash, err := parseHexBytesUrlParam(context, common.UrlParameterBlockHash)
+	if err != nil {
+		return core.OptionalUint64{}, nil, fmt.Errorf("%w for block hash", err)
+	}
+
+	return blockNonce, blockHash, nil
+}
+
 func returnBadRequest(context *gin.Context, errScope string, err error) {
 	message := fmt.Sprintf("%s: %s", errScope, err)
 	shared.RespondWith(context, http.StatusBadRequest, nil, message, data.ReturnCodeRequestError)
 }
 
-func returnOkResponse(context *gin.Context, dataToReturn interface{}) {
-	shared.RespondWith(context, http.StatusOK, gin.H{"data": dataToReturn}, "", data.ReturnCodeSuccess)
+func returnOkResponse(context *gin.Context, dataToReturn interface{}, blockInfo interface{}) {
+	shared.RespondWith(context, http.StatusOK, gin.H{"data": dataToReturn, "blockInfo": blockInfo}, "", data.ReturnCodeSuccess)
```

Comment:
isn't `blockInfo` returned for 400-code responses as well?

Reply:
from node, blockInfo is only returned on success